plugins/UFPWriter/UFPWriter.py

```diff
@@ -23,7 +23,7 @@
 
 class UFPWriter(MeshWriter):
     def __init__(self):
-        super().__init__(add_to_recent_files = False)
+        super().__init__()
```

Comment:
GCodeWriter and GCodeGzWriter both have this, why should UFPWriter be different?

Reply:
Oh, yes, the add to recent should be only in the reader. Reverted on 66a2425b863c8e2a0cd8dee78b30afba0aead164